src/custom/pages/Account/Tokens/TokensOverview.tsx

```diff
@@ -37,7 +37,7 @@ export default function TokensOverview() {
         </AccountHeading>
         <AccountCard>
           {favouriteTokens.length > 0 ? (
-            <TokensTable balances={balances} tokensData={favouriteTokens} />
+            <TokensTable loadingRows={4} balances={balances} tokensData={favouriteTokens} />
```

Comment:
magic number. also thought this was 6?

Reply:
I don't know if we will remove this table but rn it looks better with a smaller number